datafusion/functions/src/string/mod.rs

```diff
@@ -149,6 +149,9 @@ pub mod expr_fn {
     ),(
         uuid,
         "returns uuid v4 as a string value",
+    ), (
+        contains,
+        "Return true if search_string is found within string.",
```

Comment:
I think we should also mention here that search_string is treated like a regex

Reply:
sure